### PR TITLE
reuse_improve

### PR DESCRIPTION
**Pull request by juli-a-ko**
# Closes # (if applicable).

## Checklist

- [x] I consent to the release of this PR's code under the GPLv3 license and non-code contributions under CC0-1.0 and CC-BY-4.0.
- [ ] I tested my contribution locally and it seems to work fine.
- [ ] Code and workflow changes are sufficiently documented.
- [ ] Newly introduced dependencies are added to `envs/environment.yaml` and `envs/environment.docs.yaml`.
- [ ] Changes in configuration options are added in all of `config.default.yaml` and `config.tutorial.yaml`.
- [ ] Add a test config or line additions to `test/` (note tests are changing the config.tutorial.yaml)
- [ ] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [ ] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -14,7 +14,3 @@ License: CC-BY-4.0
 Files: data/*
 Copyright:  The PyPSA-Earth and PyPSA-Eur Authors
 License: CC-BY-4.0
-
-Files: .github/*
-Copyright:  The PyPSA-Earth and PyPSA-Eur Authors
-License: CC0-1.0
```
Comment (ekatef) [inline .reuse/dep5]: @juli-a-ko, after giving some thoughts to dep5, I fear that `.github/*` files should be included there. The reason is templates which are placed in `.github` directory. License headers make working with the templates difficult, which has caused need of [urgent changes](https://github.com/pypsa-meets-earth/pypsa-earth/pull/660/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3) 

If I understand properly, that mean we have to leave `.github/*` in `dep5`. Could you please double-check this logic and adjust the PR accordingly?

**Comment by ekatef** [comment]
@juli-a-ko thanks a lot for the PR! Looks great, except a question regarding `dep5`. Would be very grateful if you could address it :)

**Comment by juli-a-ko** [comment]
Ок, I left .github/* directory in dep5 file

**Review by ekatef — APPROVED** [review]

**Comment by ekatef** [comment]
> Ок, I left .github/* directory in dep5 file
Super, thanks a lot for the discussion and making the changes!

@pz-max, my feeling is that this PR ready for your review

**Review by pz-max — APPROVED** [review]
Looks good. This completes now the REUSE compatibility implementation. Thank you @juli-a-ko. It's good to have now at least three REUSE experts. Also, welcome to the contributor list :1st_place_medal: :100: Feel free to merge @ekatef

**Comment by ekatef** [comment]
Thanks a lot @juli-a-ko and @pz-max! :)

**Pull request merged**